src/pages/docs/infrastructure/deployment-targets/azure/cloud-service-targets/index.md

```diff
@@ -31,7 +31,7 @@ To create an Azure Cloud Service target within Octopus:
 
 - Go to **Infrastructure ➜ Deployment Targets ➜ Add Deployment Target**.
 - Select **Azure Cloud Service** from the list of available targets and click _Next_.
-- Fill out the necessary fields, being sure to provide a unique role that clearly identifies your Azure Cloud Service target.
+- Fill out the necessary fields, being sure to provide a unique target tag (formerly target role) that clearly identifies your Azure Cloud Service target.
```

Comment:
What's the reason to specify "formerly" here? I think we should be fine with leaving just "target tag".

Reply:
The reason for this was that the outdated UI screenshot still contains references to target roles 

But on second consideration, I don't think this will be a major point of confusion for the user for all the reasons we've talked about before

I've removed the "formerly target roles" language on this and other pages changed in this PR